spec/models/good_spec.rb

```diff
@@ -4,4 +4,40 @@
   describe 'associations' do
     it { should belong_to(:basket) }
   end
+
+  describe 'tax calculation' do
```

Comment:
When unit testing a model, a good practice to describe the methods. More on that here: https://www.betterspecs.org/#describe